components/molecule/autosuggest/src/index.js

```diff
@@ -0,0 +1,245 @@
+/*
+ TODO
+ - comprobar que el foco de la lista va bien
```

Comment:
Write all your comments in english

Reply:
okay, those are meant just to follow up next actions.  But I will write down in english from now on